lib/install.js

```diff
@@ -150,16 +154,21 @@ function install(opts, cb) {
 function reloadProxy(opts, cb) {
     assert.object(opts, 'opts');
     assert.string(opts.basePath, 'opts.basePath');
+    assert.optionalString(opts.excludePath, 'opts.excludePath');
     assert.string(opts.method, 'opts.method');
     assert.object(opts.req, 'opts.req');
     assert.object(opts.res, 'opts.res');
     assert.string(opts.routeName, 'opts.routeName');
     assert.string(opts.sourceFile, 'opts.sourceFile');
     assert.func(cb, 'cb');
 
+    var excludeFullpath = opts.basePath + '/' + opts.excludePath;
+
     // clear require cache for code loaded from a specific base dir
     Object.keys(require.cache).forEach(function (cacheKey) {
-        if (cacheKey.indexOf(opts.basePath) !== -1) {
+        if (opts.excludePath && cacheKey.indexOf(excludeFullpath) !== -1) {
```

Comment:
If you return you don't need else

Reply:
pwned

Reply:
Actually i feel like `else if` is a bit clearer.